docs/operations/debug.md

```diff
@@ -0,0 +1,59 @@
+# Debug
+This page contains instructions on how to debug Kubevirt.
+
+This is useful to both Kubevirt developers and advanced users that would like to gain deep understanding on what's
+happening behind the scenes.
+
+## Log Verbosity
```

Comment:
In this section, it would be nice to have a guideline (I assume there is guidance in code for devs creating log entries) explaining the full range of debug numbers and what conceptual level of debugging output they might correspond to, e.g. the Unix/Linux syslog of Crit, Error, Warn, Info, Debug. Also, is 6 the maximum used in code? What if somebody sets it to 9 (or 99)?

Reply:
I totally agree in principle, but in practice there is no clear definition of the exact meaning of the values or the min/max number that can be used. It is clear that higher value means more verbosity, but that's that. I agree we should properly define it in the future.

Reply:
That's surprising, but yes, it would seem out of scope for this PR then.